### ISSUE TITLE
Support for wildcard domains

### ISSUE THREAD

**Issue by pc-magas** [Aioros/livehosts#10]
In my case whilst I develop apps I need to resolve wildcard domains for example:

```
*.example.com
```

Because my application requires dynamic subdomains.
How I can configure that using livehosts?

**Comment by SebTardif** [comment]
+1 Same here. We have wildcard dns, and wildcard ssl certificate, and want to use this plug-in to match that.

**Comment by molitar** [comment]
yeah it's useless because all the ad sites have wildcard sub-domains.. Uninstalling as it does nothing useful.
